saleor/static/dashboard-next/orders/components/OrderHistory/OrderHistory.tsx

```diff
@@ -8,33 +7,89 @@ import Skeleton from "../../../components/Skeleton";
 import {
   Timeline,
   TimelineAddNote,
-  TimelineNode,
+  TimelineEvent,
   TimelineNote
-} from "../../../components/Timeline/Timeline";
+} from "../../../components/Timeline";
 import i18n from "../../../i18n";
 import { OrderEvents } from "../../../types/globalTypes";
+import { OrderDetails_order_events } from "../../types/OrderDetails";
 
 export interface FormData {
   message: string;
 }
 
+const getEventMessage = (event: OrderDetails_order_events) => {
+  switch (event.type) {
+    case OrderEvents.CANCELED:
+      return i18n.t("Order has been cancelled", {
+        context: "order history message"
+      });
+    case OrderEvents.EMAIL_SENT:
+      return i18n.t("An e-mail has been sent to customer", {
```

Comment:
There are four types of email types and we should display a more specific message depending on that type. The types and messages are:
PAYMENT - "Payment confirmation has been sent to customer"
SHIPPING = "Shipping details has been sent to customer"
ORDER = "Order confirmation has been sent to customer"
FULFILLMENT = "Fulfillment confirmation has been sent to customer"